src/store/channels-list/saga.createConversation.test.ts

```diff
@@ -0,0 +1,241 @@
+import { expectSaga, testSaga } from 'redux-saga-test-plan';
+import * as matchers from 'redux-saga-test-plan/matchers';
+
+import { createConversation as createConversationApi, uploadImage as uploadImageApi } from './api';
+
+import {
+  createConversation,
+  createOptimisticConversation,
+  sendCreateConversationRequest,
+  receiveCreatedConversation,
+} from './saga';
+
+import { RootState, rootReducer } from '../reducer';
+import {
+  ConversationStatus,
+  GroupChannelType,
+  MessagesFetchState,
+  denormalize as denormalizeChannel,
+  normalize as normalizeChannel,
+} from '../channels';
+import { setactiveConversationId } from '../chat';
+
+const MOCK_CHANNELS = [
+  { name: 'channel 1', id: 'channel_0001', url: 'channel_0001', icon: 'channel-icon', hasJoined: false },
+  { name: 'channel 2', id: 'channel_0002', url: 'channel_0002', icon: 'channel-icon', hasJoined: false },
+  { name: 'channel 3', id: 'channel_0003', url: 'channel_0003', icon: 'channel-icon', hasJoined: false },
+];
+
+describe('channels list saga', () => {
+  describe(createConversation, () => {
+    it('creates an optimistic conversation', async () => {
+      testSaga(createConversation, ['user-1'], 'name', { name: 'file' } as File)
+        .next()
+        .call(createOptimisticConversation, ['user-1'], 'name', { name: 'file' } as File)
+        .finish();
+    });
+
+    it('sets the optimistic conversation to the active one', async () => {
+      testSaga(createConversation, ['user-1'], 'name', { name: 'file' } as File)
+        .next()
+        // .call(createOptimisticConversation)
```

Comment:
are these comments here for info purposes, sort of like helper comments? so `.next({ id: 'optimistic-id' }))`  is `.call(createOptimisticConversation)`?

Reply:
Yeah, since these are specific unit tests that rely on the exact order of calls these were just to help you know which calls we're expecting but aren't really critical to the test. I'm not sure I like these tests as individual tests so I might just collapse them all down to a single test and then these would go away.